content/pt-br/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,528 @@
+
+{{< note >}} Se o último elemento do caminho for uma substring do último elemento no caminho da solicitação, não é uma correspondência (por exemplo: `/foo/bar` corresponde a `/foo/bar/baz`, mas não corresponde a `/foo/barbaz`). {{< /note >}}
+
+### Exemplos
+
+| Tipos   | Caminho(s)                         | Caminho(s) de solicitação               | Correspondências?                           |
+|---------|----------------------------------|-------------------------------|------------------------------------|
+| Prefixo | `/`                              | (todos os caminhos)           | Sim                                |
+| Exato   | `/foo`                           | `/foo`                        | Sim                                |
+| Exato   | `/foo`                           | `/bar`                        | Não                                |
+| Exato   | `/foo`                           | `/foo/`                       | Não                                |
+| Exato   | `/foo/`                          | `/foo`                        | Não                                |
+| Prefixo | `/foo`                           | `/foo`, `/foo/`               | Sim                                |
+| Prefixo | `/foo/`                          | `/foo`, `/foo/`               | Sim                                |
+| Prefixo | `/aaa/bb`                        | `/aaa/bbb`                    | Não                                |
+| Prefixo | `/aaa/bbb`                       | `/aaa/bbb`                    | Sim                                |
+| Prefixo | `/aaa/bbb/`                      | `/aaa/bbb`                    | Sim, ignora a barra final          |
+| Prefixo | `/aaa/bbb`                       | `/aaa/bbb/`                   | Sim, combina com a barra final     |
+| Prefixo | `/aaa/bbb`                       | `/aaa/bbb/ccc`                | Sim, corresponde ao subcaminho     |
+| Prefixo | `/aaa/bbb`                       | `/aaa/bbbxyz`                 | Não, não corresponde ao prefixo da string   |
+| Prefixo | `/`, `/aaa`                      | `/aaa/ccc`                    | Sim, corresponde ao prefixo `/aaa` |
+| Prefixo | `/`, `/aaa`, `/aaa/bbb`          | `/aaa/bbb`                    | Sim, corresponde ao prefixo `/aaa/bbb` |
+| Prefixo | `/`, `/aaa`, `/aaa/bbb`          | `/ccc`                        | Sim, corresponde ao prefixo `/`    |
+| Prefixo | `/aaa`                           | `/ccc`                        | Não, usa o backend padrão          |
+| Misto   | `/foo` (Prefixo), `/foo` (Exato) | `/foo`                        | Sim, prefere o exacto              |
```

Comment:
I wouldn't translate the first column (and other references to it in other columns), since it refers to a specific value that will be used in a YAML configuration.

Reply:
Sorry about that.